normits_demand/models/forecasting/EDGE_growth/ticket_splits.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+"""
+Module to process ticket type splitting factors.
+
+Splits by tag flow and purpose exist, which need to be converted to splits by station pair.
+"""
+# Built-Ins
+import pickle
+
+# Third Party
+import pandas as pd
+import numpy as np
+import caf.toolkit as ctk
+
+# Local Imports
+# pylint: disable=import-error,wrong-import-position
+# Local imports here
+from normits_demand.models.forecasting.edge_growth import utils
+from normits_demand.models.forecasting import forecast_cnfg
+from normits_demand.utils import file_ops
+# pylint: enable=import-error,wrong-import-position
+
+# # # CONSTANTS # # #
+
+# # # CLASSES # # #
+
+# # # FUNCTIONS # # #
+def append_dist(row):
+    """
+    Convert TAG_NonDist to TAG_Flow.
+
+    This function is designed to applied to a dataframe.
+    Parameters
+    ----------
+
+    row: The row this is applied to.
+    Returns
+    -------
+
+    An altered row.
+    """
+    if row["Distance"] < 25:
+        return row["TAG_NonDist"].lower() + " <25 miles"
+    elif row["Distance"] < 100:
+        return row["TAG_NonDist"].lower() + " 25 to 100 miles"
+    elif row['TAG_NonDist'].lower().endswith('london'):
+        return row["TAG_NonDist"].lower() + " 100 + miles"
+    else:
+        return row['TAG_NonDist'].lower() + " 100 + miles - adjusted"
+    
+def add_distance_band_tag_flow(mx_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Same as above, this looks like it would be much easier if it was just an input file doing the mapping!